.github/workflows/release-integrations.yml

```diff
@@ -4,6 +4,11 @@ on:
     branches:
       - main
   workflow_dispatch:
+    inputs:
+      LATEST:
```

Comment:
should be always the latest no?

Reply:
when im building manually i want the option to build images for development